src/content/en/2020/fonts.md

```diff
@@ -68,51 +64,88 @@ The single top country is South Korea, which is not all that surprising given th
+It was interesting to note the differences in speed for sites using the various free/open source and commercial services. When looking at First Content Paint (FCP) and Last Content Paint (LCP) times, sites using Google Fonts are roughly in the middle, but generally a bit slower than the median value. The fastest sites in the dataset are Shopify and Wix (serving assets from `parastorage.com`), and it might be presumed they focus on a small number of highly optimized files. Google on the other hand is also serving web fonts globally of widely varying sizes (due to language), likely resulting in slightly slower median times.
 
-#### Local isn't always better
+{{ figure_markup(
+  image="fonts-median-lcp-of-sites-using-hosted-fonts.png",
+  caption="Median LCP of sites using hosted fonts.",
+  description="A bar chart showing the median FCP of desktop and mobile sites using various font hosts. `cdn.shopify.com` is fastest at 3,335 on desktop and 8,401 on mobile, `fonts.shopifycdn.com` is at 3,224 and 8,531 respectively, `netdna.bootstrapcdn.com` is at 3,910 and 8,183, `maxcdn.bootstrapcdn.com` is at 4,240 and 8,530, `cdnjs.cloudflare.com` is at 4,105 and 8,730, `use.fontawesome.com `is at 4,519 and 9,166, `fonts.gstatic.com` is at 4,878 and 9,558, `cdn.jsdelivr.net` is at 5,368 and 10,646, `static.parastorage.com` is at 4,322 and 11,813, `use.typekit.net` is at 4,700 and 12,552",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=2012634758&format=interactive",
+  sql_file="04_05.web_font_usage_breakdown_with_fcp.sql"
+  )
+}}
+
+When viewing commercial services such as Adobe (`use.typekit.net`) or Monotype (`fast.fonts.com`) it is interesting to note that on desktop they tend to be as fast or slightly faster than Google Fonts, but are noticeably slower on mobile. Conventional wisdom has generally held that the tracking scripts used by those services substantially slow them down, but that is apparently less an issue today than it has been in years past.
 
-The use of local is [controversial](https://bramstein.com/writing/web-font-anti-patterns-local-fonts.html), as it can save bytes, but it can also yield bad results if the locally installed version of the font is outdated. As of [November 2020](https://twitter.com/googlefonts/status/1328761547041148929?s=19), Google Fonts has moved to using local only for Roboto on mobile platforms, otherwise the font is always fetched over the network.
+{# TODO authors - Do we want to add a note to say that we are measuring site performance and not necessarily font hoster performance? It could be that Adobe sites are all just pooor on mobile and it's nothing to do with their hosting platform #}
```

Comment:
@raphlinus / @jpamental thoughts?

Reply:
I think there is still some confusion about what was mean in the data and in the writing about 'local'. If I understand @AbbyTsai 's response when I asked, I think that the data analyisis was looking at `local()` pointers in the `@font-face` declaration, indicating a reference to a locally installed font on the user's system, versus 'self-hosted' web fonts that are on the same domain as the website (as opposed to coming from Google Fonts or Monotype/Adobe/etc).

I think that @raphlinus interpreted this to mean locally installed fonts on the user's system, but I had originally thought it was about self-hosting. So cleaning this question up is important to know which way to clean up the writing. 

I would really like to make sure we're looking at self-hosted fonts (or at least ones served from the same TLD as the site itself) versus services. That's the more pertinent question in my mind, and looking at the FCP/LCP data around which is better for overall performance is relevant to many developers, especially since the `local()` reference to installed fonts is generally going away.

Reply:
I looked at the query and confirmed it was looking at fonts hosted on this domain (note - not necessarily same TLD). I updated the query text as part of this PR to make this clearer. I've basically replaced all references to "local" in this chapter, in the queries, and in the graphs with either "self-hosted" or "local()".

So yes the current text in production is incorrect as it assumes it's talking about "local()" when in fact it's referencing data about self-hosted. This has been cleared up in this edit.

Reply:
Great. I'll look at it again this weekend, as more of what I had written in the original draft is relevant in that case. (Especially the comparison of service performance: that's long been a bit of a contentious topic, as it's often tied to performance impact of page-view tracking scripts used by services like Adobe and Monotype)

Reply:
I'm actually about to do a release now so if you're happy with the other edits are you OK merging this as is, and then coming back to it this weekend?

That way we at least fix the current issue where we are presenting incorrect data due to the local issue. Also the data on variable fonts also looks incorrect as looks like a mix up between desktop and mobile when calculating the percentage. This has also been fixed in this PR.

I will leave the TODOs in place to be addressed later, so they can easily be found.

Can either leave the unedited tag on there, or I'm also comfortable removing it, since it has been edited now – even if there is some more additional text to follow.

WDYT?

Reply:
I'm OK with it going out, but let's leave the 'unedited' tag until next week if that's ok. I want to have a chance to read over it all more closely, and we're moving tomorrow (!!!) so I don't have time to really look at it today. (only going one street over, but still... BOXES!)

Reply:
Done. Will merge this when the tests complete.

BTW, FYI I did keep your text - just moved it to it's own `local()` section: https://20201220t232911-dot-webalmanac.uk.r.appspot.com/en/2020/fonts#local-isnt-always-better. The changes in GitHub don't make that entirely clear!